ee/clickhouse/materialized_columns/test/test_columns.py

```diff
@@ -44,7 +44,15 @@ def test_get_columns_default(self):
         self.assertCountEqual(get_materialized_columns("person"), [])
         self.assertEqual(
             get_materialized_columns("session_recording_events"),
-            {("has_full_snapshot", "properties"): "has_full_snapshot"},
+            {
+                ("has_full_snapshot", "properties"): "has_full_snapshot",
```

Comment:
I'm not sure how you'll be using this in code, but if it's as a property that's programmatically queried for (like here): https://github.com/PostHog/posthog/blob/master/posthog/queries/column_optimizer/foss_column_optimizer.py#L40

AND, there's more than one column to materialise things from on the `session_recording_events` table, then you might want to update the materialised comments to be like:

`column_materializer::<column_name>::events_summary`. which can disambiguate between different columns.

I don't think this is a problem right now, since looks like there's only one column, and it doesn't appear like something we will query programmatically (i.e. given a property in a filter, figure out which column that property is coming from, and/or if there's a materialised version of the column).

So, all good, just a heads up for the future :)